### ISSUE TITLE
Error: Failed to install 'covidseir

### ISSUE THREAD

**Issue by ZachLucasDev** [seananderson/covidseir#6]
I followed the instructions to install the covidseir package. The previous statement regarding the pkgbuild printed TRUE. 

Error: Failed to install 'covidseir' from GitHub:
  (converted from warning) installation of package ‘C:/************/AppData/Local/Temp/Rtmps1YOPc/file40403aff55f2/covidseir_0.0.0.9010.tar.gz’ had non-zero exit status

Does anyone know what could be the cause for this error? The Stan I installed seems to be running fine. 
I am running the latest versions of Rstudio (1.3.1093) and R (version 4.0.3 (2020-10-10)).

Any help would be appreciated. 
Thank you.

**Comment by seananderson** [comment]
I’m not sure. Did you follow the instructions here? https://github.com/stan-dev/rstan/wiki/Installing-RStan-from-source-on-Windows 

They appear to have just updated that recently. The previous configuration section is gone. 

Being able to build a package from source that uses rstan is different than being able to use rstan and having the pkgbuild TRUE test. The latter two are necessary but not sufficient.

**Comment by ZachLucasDev** [comment]
Yes I did follow those instructions. Turns out it was related to my default library and a lack of permissions. Basically, because I had two libraries listed when I called .libpaths() I needed to ensure I had my common library listed as my default. For anyone who has the same issues, follow this: https://www.accelebrate.com/library/how-to-articles/r-rstudio-library?fbclid=IwAR2xV7Hiub1EnQHO6Vj9wjvl1CPY_fAAcD73DVA7PB-DuGuHR7jk42mMZQA

Thank you for follow up
